apps/communications/dialer/style/suggestion.css

```diff
@@ -7,6 +7,12 @@
   width: 100%;
   color: #033B37;
 }
+html[dir="ltr"] .suggestion-bar {
+  margin-left: -1.5rem;
```

Comment:
Why do these need different numbers? We should figure out why, and if possible, use ```margin-start: -1.5rem``` or some such, instead.